### ISSUE TITLE
Improve read me

### ISSUE THREAD

**Issue by Ghislain1** [Ghislain1/CreditCardFraudDetection#1]
2. Credit Card Fraud Detection
Language: R or Python
Data set: Data on the transaction of credit cards is used here as a data set.
Source code: [Credit Card Fraud Detection Using Python](https://www.kaggle.com/renjithmadhavan/credit-card-fraud-detection-using-python)
[Credit card fraud](https://builtin.com/data-science/data-science-cybersecurity) is more common than you think, and lately, they’ve been on the rise. We’re on the path to cross a billion credit card users by the end of 2022. But thanks to the innovations in technologies like artificial intelligence, machine learning and data science, credit card companies have been able to successfully identify and intercept these frauds with sufficient accuracy.

Simply put, the idea behind this is to analyze the customer’s usual spending behavior, including mapping the location of those spendings to identify the fraudulent transactions from the non-fraudulent ones. For this project, you can use either [R](https://builtin.com/learn/tech-dictionary/r) or [Python](https://builtin.com/data-science/enumerate-zip-sorted-reversed-python) with the customer’s transaction history as the data set and ingest it into [decision trees](https://builtin.com/data-science/classification-tree), [artificial neural networks](https://builtin.com/machine-learning/nn-models), and [logistic regression](https://builtin.com/data-science/guide-logistic-regression-tensorflow-20). As you feed more data to your system, you should be able to increase its overall accuracy.

-  https://builtin.com/data-science/data-science-project.
-  https://www.kaggle.com/code/renjithmadhavan/credit-card-fraud-detection-using-python/input
